File: examples/java/local/app/src/main/java/com/mongodb/realm/examples/model/java/Kid.java
package com.mongodb.realm.examples.model.java;

import com.mongodb.realm.examples.model.kotlin.Frog;

import io.realm.RealmList;
import io.realm.RealmObject;

// :code-block-start: one-to-many-relationship
public class Kid
        extends RealmObject {
    public RealmList<Frog> frogs =
            new RealmList<Frog>();
}
// :code-block-end:

File: examples/java/local/app/src/main/java/com/mongodb/realm/examples/model/kotlin/Kid.kt
package com.mongodb.realm.examples.model.kotlin

import io.realm.RealmList
import io.realm.RealmObject

// :code-block-start: one-to-many-relationship
class Kid : RealmObject() {
    var frogs = RealmList<Frog>()
}
// :code-block-end:
